### PR TITLE
Add conventional commits note to contributing.md

### LINKED ISSUE #15927: Add note for conventional commit messages to contributing.md
#### Issue Description

I was a nice step to enforce conventional commit messages at https://github.com/freeCodeCamp/freeCodeCamp/pull/15643.
I feel that new contributors, or even some existing contributors won't be aware about this unless they get a warning while committing something.
Can we have a section or note to this in our Contributing.md ?

If yes, I would like to update the same.

### PR DESCRIPTION
**Pull request by prateekgoel**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [ ] Tested changes locally.
- [x] Closes currently open issue: Closes #15927  

#### Description

Added note for conventional commit messages in Contributing.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -271,6 +272,19 @@ If you have made more than one commit, then you will need to _squash_ your commi
...
+The commit message should be structured as follows:
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer]
+```
+Learn more at [Conventional Commits](http://conventionalcommits.org).
+
```
Comment (dhcodes) [inline CONTRIBUTING.md]: I would add:

"For help writing your commit message, execute `npm run commit` from the command line and the [commitizen](https://github.com/commitizen/cz-cli) CLI tool will assist you in creating a conventional commit message."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ Working on your first Pull Request? You can learn how from this *free* series [H
 - [Make Changes](#make-changes)
 - [Run The Test Suite](#run-the-test-suite)
 - [Squash Your Commits](#squash-your-commits)
+- [Commit messages](#commit-messages)
```
Comment (dhcodes) [inline CONTRIBUTING.md]: For the sake of consistency, I guess this should be `Commit Message` instead of `Commit messages`

**Review by dhcodes — CHANGES_REQUESTED** [review]
I left some suggestions. Good work @prateekgoel

File: CONTRIBUTING.md (hunk 1 above)
Comment (prateekgoel) [reply CONTRIBUTING.md]: Thanks @dhcodes . I will do the changes.

**Comment by prateekgoel** [comment]
@dhcodes Did the changes.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -26,6 +26,7 @@ Working on your first Pull Request? You can learn how from this *free* series [H
 - [Make Changes](#make-changes)
 - [Run The Test Suite](#run-the-test-suite)
 - [Squash Your Commits](#squash-your-commits)
+- [Commit message](#commit-message)
```
Comment (dhcodes) [inline CONTRIBUTING.md]: @prateekgoel can you change this to `Commit Message`?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -271,6 +272,21 @@ If you have made more than one commit, then you will need to _squash_ your commi
 
 To do this, see [Squashing Your Commits](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231).
 
+### Commit message
```
Comment (dhcodes) [inline CONTRIBUTING.md]: Also, here. Please change to `Commit Message` instead of `Commit message`.

**Comment by QuincyLarson** [comment]
@dhcodes thanks for QA'ing this. Please merge when it looks ready.

@prateekgoel Thanks for the PR!

File: CONTRIBUTING.md (hunk 4 above)
Comment (prateekgoel) [reply CONTRIBUTING.md]: Changes done.

**Comment by prateekgoel** [comment]
Anytime @QuincyLarson .

**Review by dhcodes — APPROVED** [review]

**Comment by dhcodes** [comment]
**LGTM** Thanks for making those minor updates @prateekgoel and thanks for contributing as always :+1: :100: :taco:

**Comment by prateekgoel** [comment]
My pleasure @dhcodes :smiley:

**Pull request merged**